docs/examples/quickstart/cythonize/integrate_cy.pyx

```diff
@@ -2,6 +2,8 @@ def f(double x):
     return x ** 2 - x
 
 
+
+
```

Comment:
?

Reply:
Probably makes space for decorators in the Python version?

Reply:
Yes I have tried to align the two source code as requested in issue.

Reply:
hmm it's 1) out of the scope of this tutorial, 2) second file already was aligned, look [here](https://github.com/matusvalo/cython/blob/b1067464260957c69d6756e77a726c71183d2569/docs/examples/quickstart/cythonize/integrate.py) and [here](https://github.com/matusvalo/cython/blob/b1067464260957c69d6756e77a726c71183d2569/docs/examples/quickstart/cythonize/integrate_cy.pyx)

Reply:
so i still don't understand

Reply:
I doublechecked that and it was introduced by mistake. Fixed.